pilot/pkg/networking/networking.go

```diff
@@ -118,6 +120,9 @@ type MutableObjects struct {
 
 	// FilterChains is the set of filter chains that will be attached to Listener.
 	FilterChains []FilterChain
+
+	// Cluster is the set of clusters that will be attached to Cluster.
```

Comment:
Sorry, a bit confused. How is a cluster attached to a cluster?

Reply:
oops, sorry, should be filters attached to cluster..